tinygrad/nn/state.py

```diff
@@ -110,7 +110,7 @@ def persistent_load(self, pid): return pid
     for n in myzip.namelist():
       if n.startswith(f'{base_name}/data/'):
         with myzip.open(n) as myfile:
-          offsets[n.split("/")[-1]] = myfile._orig_compress_start # type: ignore
+          offsets[n.split("/")[-1]] = myfile.tell()
```

Comment:
need feedback here: 
_orig_compress_start is either None or `.tell()` depending on if the file is seekable. calling `.open` successfully on the line above at 112 sets seekable to true. So, this change should be sufficient, and passes mypy checks. Any thoughts?
https://android.googlesource.com/platform/prebuilts/clang/host/windows-x86.git/+/4a36d349eb0029b267457c75da8ba7a27beea1ed/clang-r433403/python3/Lib/zipfile.py#817